guide/web-streaming.md

```diff
@@ -15,6 +15,19 @@ except that the `X3D` file and the animation are sent on the fly to the Web brow
 
 %end
 
+The web interface displays a toolbar with the following actions:
+* Simulation time: this item indicates the current simulation time on the first line of the simulation element. The second line is currently not used and indicates how long the simulation will run.
```

Comment:
I know. I just wanted to point out that this phrase looks strange. When you say "it's not used and indicates..." seems contradictory. Either it's not used, period, or it's used and it indicates something... But this is not very important...